controller/responseTypeController.js

```diff
@@ -0,0 +1,39 @@
+const ResponseType = require('../model/responseTypeModel');
+const HttpStatus = require('http-status-codes');
+
+
+exports.getAll = function(req, res) {
+    
+    ResponseType.find().then( responseTypes => {
+        if(responseTypes) {
+            return res.status(HttpStatus.OK).json(responseTypes);
+        } else {
+            return res.status(HttpStatus.NOT_FOUND).json({
+                status: 'Error',
+                message: 'Not found.'
+            });
+        }
+    }).catch(err => {
+        return res.status(HttpStatus.INTERNAL_SERVER_ERROR);
+    });
+
+};
+
+exports.getByValue = function(req, res) {
+    const valueCode = req.params.id;
```

Comment:
It makes more sense to call the `parameter` value not `id`

Reply:
Is this `getByValue` or `getById`? Because as I am seeing, you are taking the `id` from the params. If it is the first, I aggree with @zirlman. If it is renamed, look into object property shorthand for `line 25`.

Reply:
Agree with the reviews above, if the function is getting response types by value then the url param should be called 'value'.